ballerina/Dependencies.toml

```diff
@@ -186,9 +186,6 @@ version = "0.0.0"
 dependencies = [
 	{org = "ballerina", name = "jballerina.java"}
 ]
-modules = [
```

Comment:
Is there a reason to include `lang.runtime` dependency when adding symbolic links?

Reply:
It is removed from Automated TOML file update process and no relation with the "Add Symbolic Links" commit.